src/libs/azure-utils.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash/fp'
+
+
+export const defaultAzureMachineType = 'Standard_DS1_v2'
+export const defaultAzureDiskSize = 50
+export const defaultAzureRegion = 'eastus'
+
+export const defaultAzureComputeConfig = {
+  machineType: defaultAzureMachineType,
+  diskSize: defaultAzureDiskSize,
+  region: defaultAzureRegion
+}
+
+//TODO: this should be fleshed out once azure region isn't limited to workspace region
+export const azureRegions = { eastus: { label: 'East US' } }
+export const getRegionLabel = label => _.has(label, azureRegions) ? azureRegions[label].label : 'Unknown azure region'
```

Comment:
Since you're getting the label, I think the argument is a key:
```suggestion
export const getRegionLabel = key => azureRegions?.[key].label || 'Unknown azure region'
```